src/geometry/PrimitiveGroup.ts

```diff
@@ -78,7 +78,7 @@ export default abstract class PrimitiveGroup extends GeometryGroup {
       { name: 'treeIndex', array: this.treeIndex, itemSize: 1 },
     ];
     this.hasCustomTransformAttributes = false;
-    this.data = new GeometryGroupData('Primitive', capacity, this.attributes);
+    this.data = new GeometryGroupData('Primitive', 0, this.attributes);
```

Comment:
Why is the capacity 0 when it is given above?

Reply:
The this.data parameter is overwritten by children classes. I've added a comment to make that more clear.